biomage/account/account.py

```diff
@@ -86,9 +86,7 @@ def _change_password(email, password, userpool):
             --user-pool-id "{userpool}" \
             --username "{email}" \
             --password "{password}" \
-            --permanent""".format(
-            userpool=userpool, email=email, password=password
-        ),
```

Comment:
I deleted this because we're already using f-string so we don't need to use format anymore.